libtrellis/src/PyTrellis.cpp

```diff
@@ -148,7 +181,8 @@ BOOST_PYTHON_MODULE (pytrellis)
             .def_readwrite("tiles", &Chip::tiles)
             .def_readwrite("usercode", &Chip::usercode)
             .def_readwrite("metadata", &Chip::metadata)
-            .def_readwrite("global_data", &Chip::global_data)
+            .def_readwrite("global_data_ecp5", &Chip::global_data_ecp5)
```

Comment:
This will break the nextpnr import script, I think. To avoid annoying versioning issues, can this still be called global_data? I  _think_ Boost::Python should be OK with two Python names, so both global_data and global_data_ecp5 both mapping to the global_data_ecp5 field.

Reply:
Resolved by 7428bfc.